keyboards/eyeohdesigns/babyv/keymaps/bghull/config.h

```diff
@@ -0,0 +1,3 @@
+#define AUTO_SHIFT_TIMEOUT 135
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Also, this needs an include guard, eg `#pragma once`